src/main/java/seedu/address/logic/commands/AttendanceCommand.java

```diff
@@ -23,9 +23,9 @@ public class AttendanceCommand extends Command {
             + ": Adds attendance to the person identified "
             + "by the index number used in the last person listing. "
             + "Existing attendance will not be modified.\n"
-            + "Parameters: INDEX (must be a positiv integer)"
-            + "h / [ATTENDANCE]\n"
-            + "Exmple: " + COMMAND_WORD + " 1 "
+            + "Parameters: INDEX (must be a positive integer)"
+            + "a/ [ATTENDANCE]\n"
```

Comment:
i think u should use the const prefix here to be consistent with the other command classes

Reply:
noted with thanks christel~